coral-base-list/src/scripts/BaseList.js

```diff
@@ -196,9 +196,10 @@ const BaseList = (superClass) => class extends superClass {
   /** @ignore */
   focus() {
     if (!this.contains(document.activeElement)) {
-      const items = this._getSelectableItems();
+      const items = this._getSelectableItems().filter(item => !item.hasAttribute('hidden'));
```

Comment:
hidden attribute check is already there at _getSelectableItems  , https://github.com/adobe/coral-spectrum/pull/127/files#diff-94ef24e5a35ab7c53742157552b4cbebb2e45c44a77665be5225bde41235a9d7R191 .